src/mapblock.h

```diff
@@ -498,21 +498,14 @@ class MapBlock
 
 	/*
 		Used only internally, because changes can't be tracked
+		Before use check that isValidPosition(x, y. z) returns
+		"true".
 	*/
-
 	inline MapNode &getNodeRef(s16 x, s16 y, s16 z)
 	{
-		if (!isValidPosition(x, y, z))
-			throw InvalidPositionException();
-
 		return data[z * zstride + y * ystride + x];
 	}
 
-	inline MapNode &getNodeRef(v3s16 &p)
-	{
```

Comment:
Just because it's unused <i>right now</i> does not mean this should be removed IMO

Reply:
Reinstated.

Reply:
However our coding standards are based of the Linux Kernel coding standards and these state in chapter 21 "Conditional compilation" this:

> However, if a function or variable always goes unused, delete it.

And I agree with that. Dead code is subject to bitrot and I found some large blocks wrapped in #if 0 ... #endif and some of these have questionable application to the current code.